sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DoNotHardcodeCredentials_DefaultValues.vb

```diff
@@ -19,9 +22,36 @@ Namespace Tests.Diagnostics
             Dim myPassword1 As String = Nothing
```

Comment:
Please add theese tests to reflect new regex. Both lines behaves the same in production, but the second test will fail with current code.

            foo = "Server=localhost; Database=Main; User=Admin; Password=TopSecret;;       'Noncompliant
            foo = "Server=localhost; Database=Main; User=Admin; Password   =   TopSecret;; 'Noncompliant

Reply:
Ticket created to fix this: #2783